DemoServer/Utils/Demos/DemoContainer.cs

```diff
@@ -52,24 +52,33 @@ private void ParseLanguage(DemoLanguage language, string sourceFolder)
         public static DemoContainer Initialize(Dictionary<DemoLanguage, string> demoLanguageSources, ILogger<DemoContainer> logger, Settings settings) =>
             new DemoContainer(demoLanguageSources, logger, settings);
 
-        public Demo GetDemo(DemoLanguage language, string categoryName, string demoName)
+        public Demo GetDemo(DemoLanguage language, string categoryName, string demoName, ILogger logger)
```

Comment:
You can use the injected `_logger` field instead of passing the `logger`.

Reply:
You're absolutely right. I ddn't notice the dependency injection being used in the DemoContainer class. Done.